### PR TITLE
skipper: add kubernetes-default-loadbalancer-algorithm config

### PR DESCRIPTION
**Pull request by MustafaSaber**
Make default LoadBalancer algorithm configurable

### CODE REVIEW

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -282,6 +282,12 @@ skipper_cluster_scaling_schedules: ""
 # Skipper Ingress/RouteGroup backend traffic split algorithm: traffic-predicate or traffic-segment-predicate
 skipper_ingress_backend_traffic_algorithm: "traffic-predicate"
 
+{{ if eq .Cluster.Environment "test" }}
+skipper_ingress_default_lb_algorithm: "powerOfRandomNChoices"
+{{ else }}
+skipper_ingress_default_lb_algorithm: "roundRobin"
+{{end}}
+
```
Comment (AlexanderYastrebov) [inline cluster/config-defaults.yaml:290]: Lets use the default `roundRobin` for all envs and override for test clusters via config registry.
Once it is proved to be useful we can materialize it into the defaults.

File: cluster/manifests/skipper/deployment.yaml [hunk 2]
```diff
@@ -457,6 +458,7 @@ spec:
           - "-kubernetes-backend-traffic-algorithm={{ .ConfigItems.skipper_ingress_backend_traffic_algorithm }}"
           - "-address=:9990"
           - "-wait-for-healthcheck-interval={{ .Cluster.ConfigItems.skipper_wait_for_healthcheck_interval }}"
+          - "kubernetes-default-lb-algorithm={{ .ConfigItems.skipper_ingress_default_lb_algorithm }}"
```
Comment (AlexanderYastrebov) [inline cluster/manifests/skipper/deployment.yaml]: Flag is missing a dash.
Could you please place this flag next to the previously introduced `skipper_ingress_backend_traffic_algorithm`

**Comment by AlexanderYastrebov** [comment]
> update skipper version https://github.com/zalando/skipper/compare/v0.16.111...v0.16.118

Please update PR description, I think the change does not update Skipper version.

File: cluster/config-defaults.yaml (hunk 1 above)
Comment (szuecs) [reply cluster/config-defaults.yaml:290]: No default change is good. We proofed already usefulness.

**Comment by AlexanderYastrebov** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**